capstone/src/main/java/capstone/Views/CustomerMainView.java

```diff
@@ -19,11 +20,30 @@ public void print() {
     do {
       try {
 
-        clearScreen("Customer Main");
+        clearScreen("Customer Main" + "\uD83E\uDD70");
+
+        String username = Database.CURRENT_USER.getUsername();
+        System.out.println(
+            "Welcome our distinguished customer: "
+                + ConsoleColours.GREEN_BRIGHT
+                + username
+                + ConsoleColours.RESET); //
 
         System.out.println("What would you like to do?");
+        // TODO
         System.out.println("1: Change your Password");
+        // OK
         System.out.println("2: Logout");
+        // Roy
+        System.out.println("3: Deposit");
+        // Beiji
+        System.out.println("4: Withdraw");
+        // TODO: until sql branch
```

Comment:
What are these todos

Reply:
Delete already, used to take note, thanks